<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: &quot;AlignVCenter | AlignRight&quot;;
border-top: 1px solid rgb(175, 175, 175);
border-right: 1px solid rgb(175, 175, 175);
border-left: 1px solid rgb(175, 175, 175);
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>65</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(138, 196, 207);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>65</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(138, 196, 207);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>65</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(194, 165, 146);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>133</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(194, 165, 146);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>133</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>133</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>133</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>201</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(194, 165, 146);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>201</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>201</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>201</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>269</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(194, 165, 146);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>269</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>269</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>269</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PlusMinus">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>65</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(138, 196, 207);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>337</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(138, 196, 207);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>337</y>
      <width>131</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(212, 165, 204);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>337</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	margin-right: 0px;
	border: 1px solid gray;
	border-radius: 20px;
	background-color:rgb(206, 206, 206);
}

QPushButton:pressed{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dadbde, stop:1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
